src/nvim/testdir/test_ex_mode.vim

```diff
@@ -79,4 +82,20 @@ func Test_ex_mode_errors()
   quit
 endfunc
 
+func Test_ex_mode_count_overflow()
+  " this used to cause a crash
+  let lines =<< trim END
+    call feedkeys("\<Esc>Q\<CR>")
+    v9|9silent! vi|333333233333y32333333%O
```

Comment:
I guess this causes UBSAN to report signed integer overflow which the patch was suppose to fix.